src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -567,39 +567,37 @@ JSON example:
 
 ## Fieldmap data
 
-Data acquired to correct for B0 inhomogeneities can come in different forms. The
-current version of this standard considers four different scenarios. Please note
-that in all cases fieldmap data can be linked to a specific scan(s) it was
-acquired for by filling the IntendedFor field in the corresponding JSON file.
-For example:
+Data acquired to correct for *B<sub>0</sub>* inhomogeneities can come in different forms.
+The current version of this standard considers four different scenarios:
 
-```JSON
-{
-   "IntendedFor": "func/sub-01_task-motor_bold.nii.gz"
-}
-```
+ 1. [Phase-difference map](#case-1-phase-difference-map-and-at-least-one-magnitude-image)
+ 1. [Two phase maps](#case-2-two-phase-maps-and-two-magnitude-images)
+ 1. [Direct *field mapping*](#case-3-direct-field-mapping)
+ 1. ["*PEpolar*" fieldmaps](#case-4-multiple-phase-encoded-directions-pepolar)
 
-The IntendedFor field may contain one or more filenames with paths relative to
-the subject subfolder. The path needs to use forward slashes instead of backward
-slashes. Here's an example with multiple target scans:
+These four different types of field mapping strategies can be encoded
+using the following image types:
 
-```JSON
-{
-   "IntendedFor": ["ses-pre/func/sub-01_ses-pre_task-motor_run-1_bold.nii.gz",
-                   "ses-post/func/sub-01_ses-post_task-motor_run-1_bold.nii.gz"]
-}
-```
+| **Name**         | `suffix`         | **Description**                                                                                                                                                                                                      |
+| ---------------- | ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Magnitude        | magnitude\[1,2\] | Field-mapping MR schemes such as gradient-recalled echo (GRE) generate a Magnitude image to be used for anatomical reference. Requires the existence of Phase, Phase-difference or Fieldmap maps.                    |
```

Comment:
@effigies, what should we do about the brackets? just spell out the three options?

Reply:
Oh, I think I misunderstood what was going on here. I think as long as it's clear from the text how they're used, then having two different notations seems fine as a way of flagging "check the text for details".